scripts/repos.sh

```diff
@@ -0,0 +1,160 @@
+#!/usr/bin/env bash
```

Comment:
The script was moved, but in the PR I see it as delete and add which makes it harder to compare. Is it worthwhile to diff and review in detail?

Reply:
It was moved, yes, but it has also been slimmed down considerably. Previously it was potentially also pushing (syncing) to Bitbucket, whereas now it is just clone/update from a remote. The general approach hasn't been changed though so I think you don't need to review in detail.